requirements/py37.yml

```diff
@@ -25,3 +25,7 @@ dependencies:
 # dev dependencies
   - pip
   - pre-commit
+
+# docs dependencies
+  - setuptools_scm
```

Comment:
This is more of a setup dependency even though it's used during docs building... just seems odd seeing it categorised here.

Reply:
I agree. However without it the build was unable to pick up `cf_units.__version__`.